src/addBusinessDays/test.ts

```diff
@@ -132,12 +177,19 @@ describe('addBusinessDays', function () {
     )
   })
 
-  it('if custom businessDays are Monday and Wednesday, it returns the Thursday when 2 days is added on the Sunday', function () {
+  it('if custom businessDays are Monday and Wednesday, it returns the Wednesday when 2 days is added on the Sunday', function () {
```

Comment:
question: Was this a typo? Should it also be updated on line 185 below?

Reply:
I assumed it was a typo, I couldn't figure out why the correct answer would be a non-working day. And yes, I missed updating it below.

Reply:
Oh and the 15th is indeed a Wednesday.